Makefile

```diff
@@ -150,3 +150,18 @@ OPERATOR_SDK = $(shell which operator-sdk)
 endif
 endif
 
+
+# Use this target like make sync-manifests VERSION=<version>
+.PHONY: sync-manifests
+sync-manifests:
+	@echo "Syncing manifests in bindata/assets/kueue-operator using a container"
+	@podman run --rm \
+		-v $(PWD):/workspace:Z \
+		-w /workspace \
+		python:3.11-slim \
```

Comment:
If we put this in the CI, I think we would want to use a quay or red hat registry for this.

For local development this is fine.